Python/src/DigitalTwin/DoubleUnlock/DoubleUnlockDemo.lf

```diff
@@ -96,7 +96,7 @@ reactor DoubleUnlockKeyFob(auto_lock_duration(5)) {
                     press_lock.schedule(0)
                 elif key == ord("u"):
                     press_unlock.schedule(0)
-            request_stop()
+            lf_request_stop()
```

Comment:
We have not regression tests here, but based on what I saw in `lingua-franca`, we should not make these changes in the Python target...